### PR TITLE
Update spot-controller deps to latest

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

**Comment by paweldomas** [comment]
I'm getting this error on Android:
```2019-09-26 08:43:57.659 7678-7678/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.jitsi.spot, PID: 7678
    java.lang.NoSuchMethodError: No interface method putArray(Ljava/lang/String;Lcom/facebook/react/bridge/WritableArray;)V in class Lcom/facebook/react/bridge/WritableMap; or its super classes (declaration of 'com.facebook.react.bridge.WritableMap' appears in /data/app/org.jitsi.spot-7QxRyb_W0fKuPNJfunCJBQ==/base.apk)
        at com.mackentoch.beaconsandroid.BeaconsAndroidModule.createRangingResponse(BeaconsAndroidModule.java:375)
        at com.mackentoch.beaconsandroid.BeaconsAndroidModule.access$300(BeaconsAndroidModule.java:37)
        at com.mackentoch.beaconsandroid.BeaconsAndroidModule$2.didRangeBeaconsInRegion(BeaconsAndroidModule.java:346)
        at org.altbeacon.beacon.IntentHandler.convertIntentsToCallbacks(IntentHandler.java:49)
        at org.altbeacon.beacon.BeaconLocalBroadcastProcessor$1.onReceive(BeaconLocalBroadcastProcessor.java:89)
        at androidx.localbroadcastmanager.content.LocalBroadcastManager.executePendingBroadcasts(LocalBroadcastManager.java:313)
        at androidx.localbroadcastmanager.content.LocalBroadcastManager$1.handleMessage(LocalBroadcastManager.java:121)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6898)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-09-26 08:52:16.739 8265-8265/org.jitsi.spot E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.jitsi.spot, PID: 8265
    java.lang.NoSuchMethodError: No interface method putArray(Ljava/lang/String;Lcom/facebook/react/bridge/WritableArray;)V in class Lcom/facebook/react/bridge/WritableMap; or its super classes (declaration of 'com.facebook.react.bridge.WritableMap' appears in /data/app/org.jitsi.spot-Nc_v19Jb1UW-qhGTeXNsqA==/base.apk)
        at com.mackentoch.beaconsandroid.BeaconsAndroidModule.createRangingResponse(BeaconsAndroidModule.java:375)
        at com.mackentoch.beaconsandroid.BeaconsAndroidModule.access$300(BeaconsAndroidModule.java:37)
        at com.mackentoch.beaconsandroid.BeaconsAndroidModule$2.didRangeBeaconsInRegion(BeaconsAndroidModule.java:346)
        at org.altbeacon.beacon.IntentHandler.convertIntentsToCallbacks(IntentHandler.java:49)
        at org.altbeacon.beacon.BeaconLocalBroadcastProcessor$1.onReceive(BeaconLocalBroadcastProcessor.java:89)
        at androidx.localbroadcastmanager.content.LocalBroadcastManager.executePendingBroadcasts(LocalBroadcastManager.java:313)
        at androidx.localbroadcastmanager.content.LocalBroadcastManager$1.handleMessage(LocalBroadcastManager.java:121)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6898)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)```

@saghul @zbettenbuk could you please give it a try and see if you manage to run spot-controller on your env? I've tried clearing all kinds of caches and still seeing that

**Comment by saghul** [comment]
I can give it a go next week, sure!

**Comment by jorgemarrero** [comment]
Same problem 🤔

**Comment by saghul** [comment]
Oops, I missed to test this one again. Will do shortly!

**Comment by saghul** [comment]
@jorgemarrero Are you making anyh changes to the build process? Did you make a debug or release build? Did you run on device or a simulator?

**Comment by saghul** [comment]
Ok, I could repro with a release build (not sure if relevant) on a device, but not on a simulator.

**Comment by jorgemarrero** [comment]
Sorry @saghul, I'm not really using your app. But I have the same problem. 

As I have seen the problem comes with the new version of react-native. The method putArray (and also putMap) cannot be found in [WritableNativeMap](https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/bridge/WritableNativeMap.java) class.

I guess is a problem with jetfier, but I don't sure.

**Comment by saghul** [comment]
@paweldomas it should work now, can you PTAL?

File: spot-controller/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "jitsi-meet-logger": "github:jitsi/jitsi-meet-logger#5da3033d647054457597920252b3387fa43a17fa",
         "jitsi-meet-redux": "github:jitsi/jitsi-meet-redux#bdc108e3857d5649bea16ffbf60e5a367c820b5c",
         "lodash": "4.17.15",
-        "react-native-beacons-manager": "github:jitsi/react-native-beacons-manager#86847b7de89dc3c1142d68b1ce24c9ff32d734e0",
+        "react-native-beacons-manager": "github:jitsi/react-native-beacons-manager#d9e8040082570a951f3c8a7a5c96c073821841d6",
```
Comment (virtuacoplenny) [inline spot-controller/package.json]: To confirm, using the jitsi branch is okay? I noticed a change on master of the fork. Is that change no needed?

Comment (saghul) [reply spot-controller/package.json]: Correct, it’s not needed because we now run jettify on our npm dependencies.

**Pull request merged**